### DISCUSSION TITLE
Support "Plain" SQL migration files

### DISCUSSION THREAD

**Discussion by sermojohn** [amacneil/dbmate#350] [Feature Ideas]
It is currently required to mark SQL migration files with `-- migrate:up` and `-- migrate:down`.

I guess that using SQL comments provides an area for useful features for the migration tool. However, it also couples the SQL migrations to the tool.

What if dbmate supported plain SQL files using `.up.sql` and `.down.sql` suffix, so migrations are not coupled to the dbmate syntax and use dbmate in a standard way, without using comments in the migrations. This would make SQL migration files indeed plain.

**Comment by amacneil** [comment]
I'm open to a PR supporting separate `.up.sql` / `.down.sql` files if it can be done in a clean, tested, and backwards compatible way.
